src/components/Subscriptions/SubscriptionsAutoDeposit.vue

```diff
@@ -362,19 +371,19 @@ export default {
 	},
 	apollo: {
 		query: pageQuery,
-		preFetch(config, client) {
-			return client.query({
-				query: pageQuery
-			});
-		},
+		preFetch: true,
 		result({ data }) {
```

Comment:
Refactored this section to match the more modern code in `SubscriptionsMonthlyGood.vue` - removed lodash get, simplified prefetch